File: grinn-web-client/src/app/slrdepartments/slrdepartment.ts
import { Slrfactory } from './slrfactory';
import { Slrbranches } from './srbranches';

export class Slrdepartment{
    id: number;
    idparent: number;
    snm: string;
    codinref: number;
    idcard: number;
    sremark: string;
    smnemocode: string;
    slevel: string;
    snmlong: string;
    dbegin: string;
    dend: string;
    dendorder: string;
    dendclose: string;
    llist: string;
    lhide: string;
    lnvd: string;
    lnoishl: string;
    idfactory: Slrfactory;
    idbranches: Slrbranches;
}

File: grinn-api-gateway/src/main/java/ru/grinncorp/api/dto/Slrdepartment.java
package ru.grinncorp.api.dto;


import lombok.Data;
import java.util.Date;

/**
 * Simple JavaBean domain object representing an slrdepartment.
 * @author agridin74
 */
@Data
public class Slrdepartment  {
    private Long id;
    private long idparent;
    private String snm;
    private Long codinref;
    private Long idcard;
    private String sremark;
    private String smnemocode;
    private String slevel;
    private String snmlong;
    private Date dbegin;
    private Date dend;
    private Date dendorder;
    private Date dendclose;
    private Character llist;
    private Character lhide;
    private Character lnvd;
    private Character lnoishl;
    private Slrfactory idfactory;
    private Slrbranches idbranches;

}
